h2o-bindings/bin/custom/python/gen_coxph.py

```diff
@@ -19,3 +19,179 @@ def _additional_used_columns(self, parms):
 Trains a Cox Proportional Hazards Model (CoxPH) on an H2O dataset.
 """
 )
+
+examples = dict(
+    training_frame="""
+>>> heart = h2o.import_file("http://s3.amazonaws.com/h2o-public-test-data/smalldata/coxph_test/heart.csv")
+>>> predictor = "age"
+>>> response = "event"
+>>> train, valid = heart.split_frame(ratios=[.8])
+>>> coxph = H2OCoxProportionalHazardsEstimator(start_column="start",
```

Comment:
These examples don't run because the object should be heart_coxph instead of coxph
(or change the heart_coxph.train and heart_coxph.scoring_history to coxph.train and coxph.scoring_history())

This is true for many of the examples.